### DISCUSSION TITLE
Timer to toggle ON/OFF every 30 minutes

### DISCUSSION THREAD

**Discussion by plbartie** [arendst/Tasmota#21109] [Q&A]
i would like to setup a timer to toggle ON/OFF every 30 min forever? I read through the rules and syntax doc but can't make sense of any of it, if someone could please help I would be forever grateful!

**Comment by barbudor** [comment]
![Screenshot_2024-04-06-10-42-13-870_com android chrome-edit](https://github.com/arendst/Tasmota/assets/2996042/fbaf818b-b026-4f44-a263-820901b7b759)

**Comment by plbartie** [comment]
> ![Screenshot_2024-04-06-10-42-13-870_com android chrome-edit](https://private-user-images.githubusercontent.com/2996042/320193374-fbaf818b-b026-4f44-a263-820901b7b759.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI0MjU5ODAsIm5iZiI6MTcxMjQyNTY4MCwicGF0aCI6Ii8yOTk2MDQyLzMyMDE5MzM3NC1mYmFmODE4Yi1iMDI2LTRmNDQtYTI2My04MjA5MDFiN2I3NTkuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MDZUMTc0ODAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWU4MDA3MjQzZDc1NDRkOGYzMDA5YjNhNjNhZWM4NzU3ZDFmYTNjZDRlYWY4Njc1MWUzNzU5NWQzZDFiZjZjZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.zDCw30dSk6mX-449YDQUfsuBEwt_7oSIFndepn5vJcw)

> i would like to setup a timer to toggle ON/OFF every 30 min forever? **I read through the rules and syntax doc but can't make sense of any of it**, if someone could please help I would be forever grateful!

I am not a programmer as i said above I still don't know how to use it.
I tried typing it into the console, Time#Minute|30 , but it's not working/

**Comment by lalo-uy** [comment]
That is only the condition of the rule. You need to put a rule on that
condition that executes a Power Togle.
Read the documentation about Rules.
